Makefile

```diff
@@ -14,7 +14,3 @@ include common/common.mk
 clean-hook: clean-versions
 
 script_env += NOT_RELEASED_VERSIONS="$(NOT_RELEASED_VERSIONS)"
-
-# Additional upgrade tests.  Not hooked into CI ATM.
-upgrade-tests: $(VERSIONS)
```

Comment:
Could you please use a separate commit for changes of this type? And also could you please add an explanation, why this part is removed, for the future reference?

Reply:
Added as new commit